megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -1611,17 +1603,19 @@ private JPanel getOverlaysPanel() {
         comps.add(checkboxEntry(planetaryConditionsShowIndicators, null));
         planetaryConditionsShowIndicators.setSelected(GUIP.getPlanetaryConditionsShowIndicators());
 
-        SpinnerNumberModel mPlanetaryConditionsBackgroundTransparency = new SpinnerNumberModel(
-                GUIP.getPlanetaryConditionsBackgroundTransparency(), 0, 256, 1);
+        SpinnerNumberModel mPlanetaryConditionsBackgroundTransparency = new SpinnerNumberModel(GUIP.getPlanetaryConditionsBackgroundTransparency(),
+              0,
+              256,
+              1);
```

Comment:
Seems like this would be cleaner and easier to read as

```
        SpinnerNumberModel mPlanetaryConditionsBackgroundTransparency = new SpinnerNumberModel(
              GUIP.getPlanetaryConditionsBackgroundTransparency(),
              0,
              256,
              1
        );
```